ntfs_forensics/src/diskpartition.cpp

```diff
@@ -337,73 +229,48 @@ void getPartInfo(PartInfoList& results) {
     for (unsigned int partIdx = 0; partIdx < volInfo.getPartCount();
          ++partIdx) {
       const TskVsPartInfo* vsPartInfo = volInfo.getPart(partIdx);
-      if (vsPartInfo != NULL) {
+      if (vsPartInfo != nullptr) {
         results.push_back({device.str(),
                            vsPartInfo->getAddr(),
                            std::string(vsPartInfo->getDesc())});
```

Comment:
I get a compiler error on this line about an inability to convert argument 1 from initializer list to a DiskPartitionInformation &&

Reply:
I couldn't reproduce this error, even after I deleted my build folder and built from scratch

Reply:
I cannot reproduce an error here either. Works for me.